mailer.rst

```diff
@@ -658,16 +658,18 @@ images inside the HTML contents::
         ->html('... <div background="cid:footer-signature"> ... </div> ...')
```

Comment:
Should we replace the ID in this code example too?

Reply:
That cid relates to the filename prop of the DataPart and symfony will replace it with a generated hash in the source. The example is correct.